hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -124,6 +132,37 @@ public void toCsvFile(ServicesState signedState, Instant when) {
 		if (exportSucceeded) {
 			tryToSign(csvLoc);
 		}
+		log.info("It took total {} Millisecond to export and sign the csv account files", (System.nanoTime() - startTime) / 1_000_000);
+	}
+
+	// TODO: add the log info to tell how long it take to export and sign the proto account balances file
+	//       as some validation info
+	//       Do the same to csv file.
+	@Override
+	public void toProtoFile(ServicesState signedState, Instant when) {
```

Comment:
nit : `when` can be renamed to `timeStamp` or similar

Reply:
Renamed